.prettierrc.json

```diff
@@ -1,6 +1,9 @@
 {
-    "trailingComma": "none",
-    "tabWidth": 4,
+    "tabWidth": 2,
+    "useTabs": false,
     "semi": true,
-    "bracketSpacing": true
+    "singleQuote": true,
```

Comment:
burde også få med dette:

"jsxSingleQuote": true,
"quoteProps": "preserve"

Reply:
Er ikke jsxSingleQuote spesifikt til React?
Og hva gjør egentlig quoteProps? Skjønte ikk helt det som stod om det i prettier dokumentasjonen. Har du et eksempel på når det er nyttig?

Reply:
åh jo, det er det kanskje😅 ups

Jeg fikk issues i alle config-filene mine, f.eks. .eslintrc.json. På save så endret den fra double quotes til single quotes, noe jeg ikke vil der. Har ikke en bedre forklaring, hehe

Reply:
Okey. 
Jeg tenkte egentlig at det kunne vært lurt å lage egne mapper for forskjellige rammeverk, også blir denne default.

Da kan vi ha f.eks. 
React/.prettierrc
Angular/.prettierrc
Vue/.prettierrc
Svelte/.prettierrc
osv...

Så blir filene minst mulig, og man slipper både kompleksitet og potensielle feil som følger av ulike prettier plugins.

Reply:
Kommer kanskje med et forslag til en prettier config for Svelte prosjekter etterhvert. Må bare prøve det ut litt i et reelt prosjekt først

Reply:
najs!

btw, sjekk eksemplet jeg nevnte med og uten "quoteProps": "preserve"

Reply:
Lagt til nå